tei_adapter/terraform/locals.tf

```
@@ -13,7 +13,8 @@ locals {
   min_capacity                     = 0
   max_capacity                     = 15
   rds_max_connections              = 45
-  tei_id_extractor_max_connections = 9
+  tei_id_extractor_max_connections = 5
+  rds_lock_timeout_seconds         = 600
```

I think you can do multiplication in Terraform expressions – let's make this a bit more obvious that it's 10 minutes:

```suggestion
  rds_lock_timeout_seconds         = 10 * 60
```